Конспект Git/GitHub

Ra1n-xD opened the pull request:
Конспект основных команд для работы с git и github

Ra1n-xD committed:
Конспект Git/GitHub 

Конспект основных команд для работы с git и github

Ra1n-xD commented:
Привет)

Ra1n-xD committed:
Delete Основные команды для работы с git и github.docx

Ra1n-xD committed:
Основные команды для работы с Git/GitHub

Ra1n-xD committed:
Update Основные команды Git_GitHub

Ra1n-xD committed:
Update Основные команды Git_GitHub

Ra1n-xD committed:
Update Основные команды Git_GitHub

Ra1n-xD committed:
Update Основные команды Git_GitHub

Ra1n-xD committed:
Update Основные команды Git_GitHub

Ra1n-xD committed:
Update Основные команды Git_GitHub

Ra1n-xD committed:
Команды Git/GitHub

Ra1n-xD commented:
Основные команды для работы с git и github

maximumJavascript commented on Основные команды Git_GitHub:
```diff
@@ -0,0 +1,23 @@
+
+
+
+  
```
необходимо добавить в конце ссылку на другую статью, например на Оглавление

maximumJavascript commented on Основные команды Git_GitHub:
```diff
@@ -0,0 +1,23 @@
+•	git config --local(global) user.email example@gmail.com – информация о прогере
+•	git status – статус гита
+•	git add -A (или названия файла) – все добавить файлы
+•	git commit -a –m ”commit” – коммит
```
`-a` опция еще `git add -A` делает перед коммитом, важная хрень). Я кстати ей не пользуюсь, а делаю отдельно git add

maximumJavascript commented on Основные команды Git_GitHub:
```diff
@@ -0,0 +1,23 @@
+•	git push -u origin master – запушить файлы в репозиторий (origin)
+•	git push – запушить файлы в репозиторий, который мы выбрали командой выше
+•	git remote set-url origin “url” – изменить репу
+•	git remote – удалить привязку к репу
```
чето не дописал)

Ra1n-xD committed:
Команды Git/GitHub

Ra1n-xD committed:
Delete Основные команды Git_GitHub

Ra1n-xD committed:
Update Основные команды для работы с git и github.md

Ra1n-xD committed:
Delete Основные команды для работы с git и github.md

Ra1n-xD committed:
Команды для работы с Git/GitHub

Ra1n-xD committed:
Delete Основные команды для работы с git и github.md

Ra1n-xD committed:
Add files via upload

Ra1n-xD committed:
Delete Основные команды для работы с git и github.md

Ra1n-xD committed:
Основные команды для работы с Git и GitHub

Ra1n-xD commented:
Добавил ссылку

Ra1n-xD committed:
Update Конспект Git и GitHub.md

1) Дописал команду в строчке "14"
2) Добавил пояснение для "origin"
3) Добавил ссылку на файл symbol

Ra1n-xD committed:
Update Конспект Git и GitHub.md
